cmd/commands/common.go

```diff
@@ -376,6 +376,30 @@ func ensureKubeContextName(flag *pflag.Flag) (string, error) {
 	return contextName, nil
 }
 
+func validateKubeContext(ctx context.Context, kubeContextName string, force bool) error {
+	if kubeContextName == "" {
```

Comment:
will be resolved later on by ap to the currentContext, therefore no validation required here